_saas-integrations/pepperjam/v1/pepperjam-v1.md

```diff
@@ -0,0 +1,124 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Pepperjam
+permalink: /integrations/saas/pepperjam
+keywords: pepperjam, integration, schema, etl pepperjam, pepperjam etl, pepperjam schema
+layout: singer
+# input: false
+
+key: "pepperjam-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "pepperjam"
+display_name: "Pepperjam"
+
+singer: true
+status-url: ""
+
+tap-name: "Pepperjam"
+repo-url: https://github.com/singer-io/tap-pepperjam
+
+this-version: "1"
+
+api: |
+  [{{ integration.display_name }} Advertiser API v20120402](https://support.pepperjam.com/s/advertiser-api-documentation){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: false 
+
+api-type: "platform.pepperjam"
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Standard"
```

Comment:
Right now this is showing as `Free` in the app.

Also, anything that's community supported (`certified: false`) will _almost always_ be `Free`.